cdap-ui/app/features/hydrator/controllers/create/toppanel-ctrl.js

```diff
@@ -74,6 +74,11 @@ class TopPanelController{
   }
   saveMetadata() {
     this.ConfigActionsFactory.setMetadataInfo(this.state.metadata.name, this.state.metadata.description);
+    if(this.state.metadata.description.indexOf('/\n/g') === -1) {
```

Comment:
isn't this condition wrong? you want to replace \n with space if you find it right?